### PR TITLE
Adding demo on how to customize the shadowing in the Shadow topic.

### LINKED ISSUE #754: Demonstration/Explanation how to set the shadows classes.
Link to [Issue 2788 in igniteui-angular](https://github.com/IgniteUI/igniteui-angular/issues/2788).

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #754

### CODE REVIEW

File: en/components/shadows.md [hunk 1]
```diff
@@ -148,4 +148,85 @@ Shadows are used to establish and maintain functional boundaries between Documen
...
+
+ ```scss
+...
+//Define the 3 elevation colors
+$color-1:#99BFAA;
+$color-2:#5C868D;
+$color-3:#5C3D46;    
+
+//Define the shadow elevations
+$shadow-elevations: igx-elevations($color-1,$color-2,$color-3);
```
Comment (simeonoff) [inline en/components/shadows.md]:
```suggestion
$shadow-elevations: igx-elevations($color-1, $color-2, $color-3);
```

File: en/components/shadows.md [hunk 2]
```diff
@@ -148,4 +148,85 @@ Shadows are used to establish and maintain functional boundaries between Documen
...
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="shadows-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">StackBlitz</button>
+</div>
+<div class="divider--half"></div>
+
+Now lets change the shadow color and the elevation level:
+
+ ```scss
+...
+//Define the 3 elevation colors
+$color-1:#99BFAA;
```
Comment (simeonoff) [inline en/components/shadows.md]:
```suggestion
$color-1: #99BFAA;
```

File: en/components/shadows.md [hunk 3]
```diff
@@ -148,4 +148,85 @@ Shadows are used to establish and maintain functional boundaries between Documen
...
+$color-2:#5C868D;
```
Comment (simeonoff) [inline en/components/shadows.md]:
```suggestion
$color-2: #5C868D;
```

File: en/components/shadows.md [hunk 4]
```diff
@@ -148,4 +148,85 @@ Shadows are used to establish and maintain functional boundaries between Documen
...
+$color-3:#5C3D46;
```
Comment (simeonoff) [inline en/components/shadows.md]:
```suggestion
$color-3: #5C3D46;
```

File: en/components/shadows.md [hunk 5]
```diff
@@ -148,4 +148,85 @@ Shadows are used to establish and maintain functional boundaries between Documen
...
+###Demo
+
+The following section demonstrates how to set various elevations to elements.
+
+Define the element that you want ot set a shadow on:
+
+```html
+...
+    <div>
+        <div class="elevation-sample elevate">
```
Comment (simeonoff) [inline en/components/shadows.md]: Why the two classes. Isn't `elevation-sample` enough?

**Comment by nrobakova** [comment]
The title of the topic should be changed to 'Elevations Settings' instead of 'Demo'
The changes should be added to the jp files also.

**Pull request merged**